art/attacks/evasion/fast_gradient.py

```diff
@@ -472,7 +472,11 @@ def _apply_perturbation(
         batch = batch + perturbation_step
         if self.estimator.clip_values is not None:
             clip_min, clip_max = self.estimator.clip_values
-            batch = np.clip(batch, clip_min, clip_max)
+            if batch.dtype == np.object:
```

Comment:
Shouldn't we use numpy?

Reply:
This change is to handle object arrays of numpy arrays.